community/kernel/src/main/java/org/neo4j/graphdb/factory/GraphDatabaseSettings.java

```diff
@@ -464,6 +501,31 @@
     public static final Setting<String> default_advertised_address =
             setting( "dbms.connectors.default_advertised_address", STRING, "localhost" );
 
+    @Description( "Create an archive of an index before re-creating it if failing to load on startup." )
+    @Internal
+    public static final Setting<Boolean> archive_failed_index = setting(
+            "unsupported.dbms.index.archive_failed", BOOLEAN, "false" );
+
+    // Needed to validate config, accessed via reflection
+    @SuppressWarnings( "unused" )
+    public static final BoltConnectorValidator boltValidator = new BoltConnectorValidator();
+
+    // Needed to validate config, accessed via reflection
+    @SuppressWarnings( "unused" )
+    public static final HttpConnectorValidator httpValidator = new HttpConnectorValidator();
```

Comment:
I would like to pull all connectors out of the kernel, but bolt being a kernel extension kind of prevents that. There are a lot of stuff in the kernel which shouldn't logically belong there but them being there now forces stuff to be there because of service loading